pkg/arvo/app/dbug.hoon

```diff
@@ -400,7 +401,7 @@
 ++  get-file-at
   |=  [base=path file=path ext=@ta]
   ^-  (unit octs)
-  ?.  ?=(?(%html %css %js %png) ext)
+  ?.  ?=(?(%html %css %js %png %woff2) ext)
```

Comment:
The fileserving logic in `/app/dbug` is a holdover from when we didn't have `/app/file-server` yet, and should be ripped out. Your change here would run into the same issue 
@joemfb described in his comments (scrying for woff2 as `@`). You're fine to just revert your changes here, the debug dashboard doesn't serve/need fonts anyway.